index.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <link rel="stylesheet" href="index.css">
+
+  <title>Hello Microverse</title>
+</head>
+
+<body>
+
+
+  <p>Hello Microverse</p>
+
```

Comment:
- [ ] Please change the p tag to header tag and add a selector to the tag as suggested in the requirement page for the project.